3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,87 @@
+//Age-ify (A future age calculator)
+let yearOfBirth = 1999;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log('you will be' + " " + age + ' yeras old in' + " " + yearFuture);
+
+// A Dog Age Calculator 
+let dogYearOfBirth = 2017
+let dogYearFuture = 2027;
+let dogYear = "dog";
+
+let shouldShowResultInDogYears = true;
+let dogAge = dogYearFuture - dogYearOfBirth;
+let humanAge = dogAge * 7;
+
+if (shouldShowResultInDogYears) {
+    console.log("your dog will be " + dogAge + " " + dogYear + " " + "years old in" + " " + dogYearFuture)
+} else {
+    console.log("your dog will be " + humanAge + " " + dogYear + " " + "human years old in" + " " + dogYearFuture)
+}
+
+//Housey pricey (A house price estimator)
+
+let peterHouse = {
+    wide: 8,
+    deep: 10,
+    high: 10,
+    gardenSize: 100,
+};
+
+let juliaHouse = {
+    wide: 5,
+    deep: 11,
+    high: 8,
+    gardenSize: 70,
+};
+
```

Comment:
👍 The separation of house properties into objects and the modular approach to calculate the price is a good design choice.